test/api/students_api_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'test_helper'
+require 'user'
+
+class StudentsApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+  include TestHelpers::AuthHelper
+  include TestHelpers::JsonHelper
+
+  def app
+    Rails.application
+  end
+
+  def test_get_students_with_authentication
+    # Create admin
+    #adminUser = FactoryBot.create(:user, :admin)
```

Comment:
Next time please do not commit code comments in your push if they do not work yet, get rid of them when pushing commit and only commit when the codes work